great-idea/contact.html

```diff
@@ -0,0 +1,88 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Great Idea! - Contact</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="css/contact.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+<body>
+
+  <div class="container">
+
+    <div class="nav-container">
+      <header class="nav-content">
+        <nav>
+          <a href="services.html" class="nav-button">Services</a>
+          <a href="" class="nav-button">Product</a>
+          <a href="" class="nav-button">Vision</a>
+          <a href="" class="nav-button">Features</a>
+          <a href="" class="nav-button">About</a>
+          <a href="contact.html" class="nav-button">Contact</a>
+        </nav>
+        <a class="logo" href="index.html"><img src="img/logo.png" alt="Great Idea! Company logo."></a>
+      </header>
+    </div>
+
+    <section class="top-content">
+        <img class="lightbulbs" src="img/contact-header.jpeg" alt="Our services header image">
+        <div class="contact-us">
+            <h1>Tell us how we're doing!</h1>
+            <p>We learn from your feedback, nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
+            </p>
+        </div>
+        <div class="vision">
+            <h1>vision</h1>
+            <p>At Great Idea! we believe that aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+            </p>
+        </div>
+        <address>
+                <h1>Contact</h1>
+                <p>123 Way 456 Street <br>
+                Somewhere, USA</p> <br>
+                <p>1 (888) 888-8888</p> <br>
```

Comment:
```suggestion
              <a href="tel:18888888888">1 (888) 888-8888</a>
```
Putting this into an anchor tag and using 'tel:PHONE-NUMBER' will make this more semantic. Your user will know that this is a phone number, overall allowing for a better UX